src/component/vocabulary/RemoveVocabularyDialog.scss

```diff
@@ -0,0 +1,19 @@
+#remove-vocabulary-dialog {
+  .modal-header {
+    background-color: var(--danger);
```

Comment:
Why is the header using `danger` color? It is quite aggressive and inconsistent with other TermIt UI